src/js/themes/base.js

```diff
@@ -198,7 +198,10 @@ export const generate = (baseSpacing = 24, scale = 6) => {
           color: 'focus',
         },
       },
-      font: { ...fontSizing(0) },
+      font: {
+        size: 'inherit',
+        height: 'normal',
```

Comment:
I don't think we want this to be `normal`, if the caller uses this in a context where the `line-height` has been explicitly set, this will override that setting. I think this wants to be `inherit` as well.